examples/svelte-custom-store/src/lib/store.ts

```diff
@@ -0,0 +1,26 @@
+import { writable } from "svelte/store";
+
+// In theory, it should be possible to remove the wxtStore.watch call 
+// and only listen to changes in the svelte store, 
+// but then the changes don't propagate from popup to content/background.
+// Improvements welcome!
```

Comment:
I don't know how `svelte/store` works, but I assume it only works for a single context. So we call `.watch` to keep this context in sync with all the other contexts.

Reply:
That could very well be

Reply:
As to your previous comment, a subscriber in svelte immediately receives the current state upon subscription (see 2nd example under [#writable](https://svelte.dev/docs/svelte-store#writable)). So in that sense the console output is not duplicate.

Other than that thank you for cleaning up and the minor refactor, looks all good to me!